verification/verify.go

```diff
@@ -33,8 +33,10 @@ type Fatal struct {
 	Retryable
 }
 
+var ErrTampered = Retryable{errors.New("Tampered")}
```

Comment:
Why's this being added?

Reply:
That error moved from epic verifier. Currently, local verification doesn't produce it, but it's used in multiple unit tests, so I left it.